src/app/zap-templates/zcl/data-model/chip/descriptor-cluster.xml

```diff
@@ -19,6 +19,7 @@ limitations under the License.
 
   <struct name="DeviceTypeStruct">
     <cluster code="0x001d"/>
+    <cluster code="0x0750"/>
```

Comment:
I don't see any basis in the spec for this.  Ecosystem information has a `list[device-type]`, but nothing defines what `device-type` is, as far as I can see.

Reply:
I chatted with Chris before making this change and confirmed that this is what he meant. [I added a comment to the spec PR](https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/9499/files#r1673156450) that I am waiting to have that land before I land this PR